elastalert/test_rule.py

```diff
@@ -185,26 +183,23 @@ def mock_elastalert(self, elastalert):
         elastalert.get_hits = self.mock_hits
         elastalert.elasticsearch_client = mock.Mock()
 
-    def run_elastalert(self, rule, conf, args):
+    def run_elastalert(self, conf, args):
         """ Creates an ElastAlert instance and run's over for a specific rule using either real or mock data. """
 
-        # Load and instantiate rule
-        load_modules(rule)
-        conf['rules'] = [rule]
-
-        # If using mock data, make sure it's sorted and find appropriate time range
-        timestamp_field = rule.get('timestamp_field', '@timestamp')
         if args.json:
```

Comment:
I'm not sure how this is supposed to work with JSON data AND multiple rules. things like timestamp_field can change per rule. I think if there are multiple rules it should not let you use json data, unless we can figure out a more elegant way to handle that.